frontend/app/(footer)/fulfilment/[fulfilmentSessionId]/[fulfilmentEntityId]/page.tsx

```
@@ -0,0 +1,365 @@
+
+        try {
+          const savedFormResponseId = await formResponderRef.current.save();
+          fulfilmentSessionEntityPageLogger.info(
+            `Form response saved with ID: ${savedFormResponseId}, fulfilmentSessionId: ${params.fulfilmentSessionId}, fulfilmentEntityId: ${params.fulfilmentEntityId}`
+          );
+
+          await handleNext();
+        } catch (error) {
+          fulfilmentSessionEntityPageLogger.error(`Error saving form and navigating to next: ${error}`);
+        }
+      };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)